test/framework/src/main/java/org/elasticsearch/transport/AbstractSimpleTransportTestCase.java

```diff
@@ -1610,7 +1610,9 @@ public void testVersionFrom1to1() throws Exception {
             assertThat(request.value2, equalTo(2));
             Version1Response response = new Version1Response(1, 2);
             channel.sendResponse(response);
-            assertEquals(version1.transportVersion, channel.getVersion());
+            // channel versions don't make sense on DirectResponseChannel
+            assertThat(channel, instanceOf(TaskTransportChannel.class));
```

Comment:
`DirectResponseChannel` now always returns `TransportVersion.CURRENT` for the channel version, rather than using the node version, so this check doesn't make sense for these two tests now.